docker-services.yml

```diff
@@ -102,3 +102,7 @@ services:
       - REDIS_HOSTS=cache
     ports:
       - "8081:8081"
+  cap-client:
+    build:
+      context: https://github.com/cernanalysispreservation/cap-client.git#docker
```

Comment:
please rerun and update with correct url. There is not `docker` branch in the official repo